packages/actor-http-fetch/lib/ActorHttpFetch.ts

```diff
@@ -58,22 +58,44 @@ export class ActorHttpFetch extends ActorHttp {
       action.init.headers = ActorHttp.headersToHash(action.init.headers);
     }
 
+    const requestInit = { ...action.init };
+
+    if (action.context.get(KeysHttp.includeCredentials)) {
+      requestInit.credentials = 'include';
+    }
+
+    const httpTimeout: number | undefined = action.context?.get(KeysHttp.httpTimeout);
+    let setupHttpTimeout: (() => void) = () => {
+      // No op by default
+    };
+    if (httpTimeout !== undefined) {
+      // Fallback to abort-controller for Node 14 backward compatibility
+      /* istanbul ignore next */
+      const AbortController = global.AbortController || await import('abort-controller');
```

Comment:
I would recommend moving this logic into [FetchInitPreprocessor.ts](https://github.com/comunica/comunica/blob/master/packages/actor-http-fetch/lib/FetchInitPreprocessor.ts) and [FetchInitPreprocessor-browser.ts](https://github.com/comunica/comunica/blob/master/packages/actor-http-fetch/lib/FetchInitPreprocessor-browser.ts), since `AbortController` is guaranteed to exist in the browser, while for Node we may indeed need a polyfill.

That reminds me, we definitely need to test this in a browser environment as well (manually, since we don't have proper unit tests for browsers yet).

Reply:
I left the logic in `ActorHttpFetch` to make the cancelation easier but added a provider for "AbortController" to `FetchInitPreprocessor`. I hope it's fine for you.